omnibus/config/projects/chef.rb

```diff
@@ -20,7 +20,7 @@
 homepage "https://www.chef.io"
 
 build_iteration 1
-build_version "12.6.0"
+build_version "12.7.0"
```

Comment:
yeah easier code, i'm probably thinking too hard about someone who bumps the VERSION file and forgets to run the rake task before committing the change...   the rake task should wind up being controlled by CI though so that doesn't happen...